### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by Tooconfident** [CodeGeekSteve/prj-rev-bwfs-tea-cozy#7]
Grade: Meets Expectations

Not bad! Your project looks pretty solid and your css shines here. You show a solid knowledge of flex and other complex css properties like z-index. You will see my line issue's are more commentary on alternative solutions to your style choices. Whether you think they are better or helpful is up to you. I hope you find them nice options to clean things up and make it DRY(Don't Repeat Yourself). 

For your html though I highly recommend revisiting it to improve the overall layout and readability. The html is quite hard to follow due to some of your id and class names as well as indentation errors. The inconsistency in indentation makes it really hard to identify child/parent hierarchy as well as which closing tags belong to which div or other element. Please use these references to get you started on learning the importance of html indentation:

https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/
https://www.w3schools.com/html/html5_syntax.asp
https://css-tricks.com/what-beautiful-html-code-looks-like/
http://htmlformatter.com/ (great resource to paste your code in and get an idea of how it should look)

**Comment by CodeGeekSteve** [comment]
Thank you! It really helps
